### ISSUE TITLE
dft-grad-lr3 fails!

### ISSUE THREAD

**Issue by raimis** [psi4/psi4#1418]
`dft-grad-lr3` test fails with the current master (64aa80aa96983bec51d40732d18f3def5eb9e844):
```
$ ctest -R dft-grad-lr3 --output-on-failure
Test project /home/raimis/psi4.git/linux_build
    Start 158: dft-grad-lr3
1/1 Test #158: dft-grad-lr3 .....................***Failed   85.00 sec
	Analytic vs Reference wB97X-D Gradients...........................PASSED
Traceback (most recent call last):
  File "/home/raimis/psi4.git/linux_build/stage/bin/psi4", line 290, in <module>
    exec(content)
  File "<string>", line 74, in <module>
  File "/home/raimis/psi4.git/linux_build/stage/lib/psi4/driver/driver.py", line 680, in gradient
    wfn = procedures['gradient'][lowername](lowername, molecule=molecule, **kwargs)
  File "/home/raimis/psi4.git/linux_build/stage/lib/psi4/driver/procrouting/proc.py", line 2084, in run_scf_gradient
    ref_wfn = run_scf(name, **kwargs)
  File "/home/raimis/psi4.git/linux_build/stage/lib/psi4/driver/procrouting/proc.py", line 2022, in run_scf
    scf_wfn = scf_helper(name, post_scf=False, **kwargs)
  File "/home/raimis/psi4.git/linux_build/stage/lib/psi4/driver/procrouting/proc.py", line 1363, in scf_helper
    disp_energy = scf_wfn._disp_functor.compute_energy(scf_wfn.molecule())
  File "/home/raimis/psi4.git/linux_build/stage/lib/psi4/driver/procrouting/empirical_dispersion.py", line 195, in compute_energy
    verbose=1)
  File "/home/raimis/psi4.git/linux_build/stage/lib/psi4/driver/qcdb/intf_dftd3/runner.py", line 136, in run_dftd3_from_arrays
    jobrec['qcvars']['CURRENT ENERGY'] = copy.deepcopy(jobrec['qcvars']['DISPERSION CORRECTION ENERGY'])

KeyError: 'qcvars'
Exit Status: infile ( 1 ); autotest ( None ); sowreap ( None ); overall ( 1 )


0% tests passed, 1 tests failed out of 1

Label Time Summary:
dft    =  85.00 sec*proc (1 test)
psi    =  85.00 sec*proc (1 test)
scf    =  85.00 sec*proc (1 test)

Total Test time (real) =  85.06 sec

The following tests FAILED:
	158 - dft-grad-lr3 (Failed)
Errors while running CTest
```

**Comment by loriab** [comment]
I suspect this was fixed in #1414 that made sure dftd3-using tests weren't run if dftd3 not available.

**Comment by dgasmith** [comment]
Passes locally, let me know if you continue to have problems.
